plugins/grpc-ecosystem/gateway/v2.15.2/Dockerfile

```
@@ -1,11 +1,18 @@
 # syntax=docker/dockerfile:1.4
 FROM golang:1.20.1-bullseye AS build
+
+COPY separate_pkg_additional_imports.patch /tmp/separate_pkg_additional_imports.patch
```

```suggestion
COPY --link separate_pkg_additional_imports.patch /tmp/separate_pkg_additional_imports.patch
```

I think it is also worth adding this after the git clone so that layer will be cached more often if we need to tweak the patch.